sdk/scheduler/src/test/java/com/mesosphere/sdk/config/ConfigurationUpdaterTest.java

```diff
@@ -261,12 +266,21 @@ public void testUserSetAtServiceLevelButNotPodLevel() throws ConfigStoreExceptio
         serviceSpecWithUser.pods(podsWithoutUsers);
 
         when(mockConfigStore.fetch(TARGET_ID)).thenReturn(serviceSpecWithUser.build());
+
         // Note: the new service spec sets pod users with a non-root user
-        ConfigurationUpdater.UpdateResult result =
-                configurationUpdater.updateConfiguration(ORIGINAL_SERVICE_SPECIFICATION_WITH_USER);
-        Assert.assertEquals(TARGET_ID, result.getTargetId());
-        // since the 2 pods don't set the user their user defaults to "root" which conflicts with the user set as noted above
-        Assert.assertEquals(2, result.getErrors().size());
+        boolean caughtError = false;
+        boolean correctErrorMessage = false;
+        ConfigurationUpdater.UpdateResult result = null;
+        try {
```

Comment:
(nit) i like the style we use here https://github.com/mesosphere/dcos-commons/blob/60e9caf214eed5bf0c61bea26afe05744ed93ae4/sdk/scheduler/src/test/java/com/mesosphere/sdk/curator/CuratorUtilsTest.java#L115-L118
use a fail() at the end of try and assert inside the catch, feels more readable. same pattern in other two places in this file.